src/main/java/com/easypost/model/PrimaryPaymentMethod.java

```diff
@@ -0,0 +1,4 @@
+package com.easypost.model;
+
+public class PrimaryPaymentMethod extends BaseCreditCard {
```

Comment:
Nit: If there's nothing new added here, we don't really need explicit classes for `PrimaryPaymentMethod` and `SecondaryPaymentMethod`, we can just use `BaseCreditCard`.
Won't block, because we can keep them in case properties diverge in the future.